content/en/learn.md

```diff
@@ -5,9 +5,14 @@ sidebar: false
 
 For the **official NumPy documentation** visit [numpy.org/doc/stable](https://numpy.org/doc/stable).
 
-Below is a curated collection of external resources. To contribute, see the [end of this page](#add-to-this-list).
+## NumPy Tutorials
+
+The official set of tutorials and educational materials by the NumPy community is [NumPy Tutorials](https://numpy.org/numpy-tutorials). The goal of this page is to provide high-quality resources by the NumPy project, both for self-learning and for teaching classes with, in the format of Jupyter Notebooks. If you’re interested in adding your own content, check the [numpy-tutorials repository on GitHub](https://github.com/numpy/numpy-tutorials).
```

Comment:
I wouldn't say _"The official set of tutorials and educational materials"_. We also have tutorials in the Sphinx docs, and those are not any less official. I don't immediately know what the best way would be to explain this split.

Reply:
That is something I'm wondering about. Are we planning on having a mirrored set of tutorials on numpy-tutorials and the sphinx docs? Should we just have a link to numpy-tutorials? Should we have two separate sets of tutorials? 

For now I'll change the wording on this and maybe we can discuss at the docs team and community meetings.

Reply:
Maybe they can be separate content types. The new notebook based ones are really focused on "perform an analysis" and have more of a story. The ones in .rst files are more like "here's to understand this one numpy concept or thingy". That's even true of the quickstart tutorial, and probably a good split to have.